kolibri/core/assets/src/kolibri_app.js

```diff
@@ -58,6 +58,12 @@ export default class KolibriApp extends KolibriModule {
     return {};
   }
 
+  handlePointerDown(event) {
```

Comment:
I think I would rather move all of this logic into the browserInfo.js module - and create a simple exportable boolean in there `mousePointerUsed` or something like that.

This can be initialized from the localStorage initially, and then we can update both the localStorage and the value when the condition below happens.